libs/stocks/feature-shell/src/lib/stocks/stocks.component.html

```diff
@@ -22,8 +19,7 @@
     <mat-select formControlName="period">
       <mat-option
         *ngFor="let timePeriod of timePeriods"
-        [value]="timePeriod.value"
-      >
```

Comment:
Performed some code indentation on this page. Its just preference of an individual though how he/she want to see it